src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,155 @@
+package duke;
+
+/**
+ * Represents a parser that parse user command.
+ */
+public class Parser {
+
+    /**
+     * Represents list of valid user commands.
+     */
+    public enum UserCommand {
+        BYE, LIST, UNMARK, MARK, TODO, DEADLINE, EVENT, DELETE, FIND
+    }
+
+    /**
+     * Handles user's input into program.
+     *
+     * @param input User input into program.
+     */
+    public static Command parse(String input) throws DukeException {
+        String[] inputList = input.split(" ");
+        Task newListItem;
```

Comment:
Your use of switch case is good! Makes your code more readable and neater

Reply:
Thanks.